### ISSUE TITLE
[Feature Request] Domain Name Support for Remote

### ISSUE THREAD

**Issue by pandalion98** [wangyu-/udp2raw-tunnel#155]
Everytime I use `-r`, I need to use an IP address.

Recently, I moved my server to my home, but it has a dynamic IP. I use a DDNS service for my domain name.

Being able to use a domain name as a remote server address would be great.

e.g. `/root/udp2raw_arm -c -a -l 0.0.0.0:1194 -r example.com:57861`

**Comment by JuniorJPDJ** [comment]
https://github.com/wangyu-/udp2raw-tunnel/pull/66
Never mind, it was pull and was not finished

**Comment by wangyu-** [comment]
You can resolve the domain simply by：

```
root@debian9:~/Desktop/UDPspeeder# nslookup g.cn|grep Add |awk '!/#/{printf$2"\n"}'
203.208.41.47
203.208.41.55
203.208.41.56
203.208.41.63
```

>Everytime I use -r, I need to use an IP address.
>Recently, I moved my server to my home, but it has a dynamic IP. I use a DDNS service for my domain name.
>Being able to use a domain name as a remote server address would be great.

In your example，use：

```
IP=`nslookup example.com|grep Add |awk '!/#/{printf$2"\n"}'|head -n1`
./udp2raw_arm -c -a -l 0.0.0.0:1194 -r $IP:57861
```

**Comment by pandalion98** [comment]
@wangyu- indeed, I use that solution, but that doesn't work with `--conf-file` unfortunately.

I use a simpler solution:
`./udp2raw_arm -c -a -l 0.0.0.0:1194 -r $(dig +short  example.com):57861`


In case `dig` is not installed (e.g. Debian): `apt-get install dnsutils`

**Comment by wangyu-** [comment]
There is a problem in #66 , so I havnt merge it yet.

I am not planning to implement this feature in the near future as its not strictly necessary. If someone implemented it correctly, I will merge the code.

**Comment by wangyu-** [comment]
>indeed, I use that solution, but that doesn't work with --conf-file unfortunately.

You can use `-r` option outside of you confile, for example:

`udp2raw -r $IP:port  --conf-file 1.conf`

Put your other options(except -r) into `1.conf`。
